source/instructors/authoring/assessments/student-submission.rst

```diff
@@ -3,51 +3,76 @@
+
+.. note:: You can customize the Submit button label to any text you prefer. In the assessment markdown, update the text to the left of "|assessment".
+    
+    .. image:: /img/guides/customizeSubmitbutton.png
+       :alt: Customize Submit Button
+       :width: 400px
 
-Each assessment type allows you to define the number of attempts students can make. On the last allowed attempt, the student will be warned that they will not be able to resubmit after that attempt.
+Each assessment type allows you to define the number of attempts students can make. Students can see the number of attempts they have left to the right of the Submit button.
 
-You can suppress the use of submit buttons for Advanced Code Test, Standard Code Test, MCQ, Fill in the Blank, Free Text and Free Text Autograde assessments. You cannot suppress the submit buttons for Parsons Puzzle assessments.
+Suppressing the Submit Button
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-This feature is handy for exams, students do not need to worry about the effect of pressing the button. They can provide a response and move on to other assessments or pages in the guide.
+You can suppress the Submit button for the following assessment types: Advanced Code Test, Standard Code Test, MCQ, Fill in the Blank, Parsons Puzzle, LLM Rubric Auto Grade, LLM Rubric, Free Text, and Free Text Autograde.
+
+This feature is useful for exams where students shouldn't need to worry about pressing a submit button. They can simply provide their response and move on to other assessments or pages.
 
 To suppress the use of the **Submit** button, go to the **settings** button in the guide and disable **Use submit buttons**.
 
   .. image:: /img/guides/globalsettings.png
      :alt: Global Settings
 
 
-Once the project is marked as complete (see below) all assessment responses are submitted automatically. All students work must be marked as complete either manually or using the automated **Mark as Complete** option on the final deadline.
+Once the project is marked as complete (see below) all assessment responses are submitted automatically. All student work must be marked as complete either manually or using the automated **Mark as Complete** option on the final deadline.
+
+
 
 Mark as Complete
 ----------------
-To suppress the student **Mark as complete** action, go to the guide **settings** (see above screenshot) and disable **Use mark as complete**.
 
-A student can proactively mark an assignment as complete. This can trigger an :ref:`assignment level autograde script <auto-grade-scripts>` and it is also displayed in the teacher dashboard for that student.
+A student can proactively mark an assignment as complete. This triggers an :ref:`assignment-level autograde script <auto-grade-scripts>` and displays the completion status in the teacher dashboard.
+
+**Advantages:**
+Instructors can grade students' work ahead of a deadline.
```

Comment:
This is a bit confusing:
"Instructors can grade students' work ahead of a deadline."
it might be clearer to say
"Instructors can grade students' work as soon as they mark it as complete."